src/server/handlers/schema.js

```diff
@@ -0,0 +1,13 @@
+import { schema } from 'normalizr';
+
+export const owner = new schema.Entity('owners');
+// XXX repository or repo?
```

Comment:
They are called 'repos' even in the UI, so I'm fine with making it consistent everywhere.